video/segment.go

```diff
@@ -15,16 +16,16 @@ func Segment(sourceFilename string, outputManifestURL string, targetSegmentSize
 	// Do the segmenting, using the local file as source
 	err := ffmpeg.Input(sourceFilename).
 		Output(
-			outputManifestURL,
+			strings.Replace(outputManifestURL, ".m3u8", "", 1)+"%d.ts",
 			ffmpeg.KwArgs{
 				"c:a":               "copy",
 				"c:v":               "copy",
-				"f":                 "hls",
-				"hls_segment_type":  "mpegts",
-				"hls_playlist_type": "vod",
-				"hls_list_size":     "0",
-				"hls_time":          targetSegmentSize,
-				"method":            "PUT",
+				"f":                 "segment",
+				"segment_list":      outputManifestURL,
+				"segment_list_type": "m3u8",
+				"segment_format":    "mpegts",
+				"segment_time":      targetSegmentSize,
+				"min_seg_duration":  "2",
```

Comment:
why 2?

Reply:
Yeah i'm not sure, I picked it for now just because I remember never seeing any lower than that in the past really. @thomshutt what do you think, is it ok to pick something pretty arbitrary like this and put it in a constant?